packages/i18nify-react/README.md

```diff
@@ -1 +1,90 @@
 # i18nify-react
+
+Welcome to i18nify-react, your new best friend for seamlessly managing i18n state in React applications! 🎉 The i18nify-react Context Provider is a simple and flexible solution for managing internationalization (i18n) state in React applications. This provider utilizes the @razorpay/i18nify-js library to handle i18n state and exposes a React Context for efficient state management across the application.
+
+## Features
+
+- **I18nProvider:** A custom React Context provider that effortlessly spreads i18n state updates across your component tree.
+- **useI18nContext Hook:** Fetch values exposed by the `I18nProvider` using this handy custom React hook.
+- **Easy Integration:** Integrate i18n state management into your React app with style and ease.
```

Comment:
change:  `I18nProvider` explaining tech details, `Easy Integration` explaining the operational details, clubbing them together seems confusing.

With `I18nProvider` and `useI18nContext` you can easily integrate i18n state management into your app. 

I18nProvider: A custom React Context provider that effortlessly spreads i18n state updates across your component tree.
useI18nContext: Fetch values exposed by the `I18nProvider` using this handy custom React hook.

something like this.

explain like this
1. What does it do
2. How does it do
3. What changes developer required todo make it work effectively

Reply:
This is just a starter readme that I've added from this [PR](https://github.com/razorpay/i18nify/pull/59) for an empty readme file. 
@RgnDunes will take some time to refine the items as suggested before. 

FYI, Above questions are addressed in below sections of the readme.
1. Introduction and Features
2. Installation
3. Usage

Reply:
Got it, can you please create the JIRA ticket for readme refinement and add that here, i will approv the PR.

Reply:
Based on this, https://razorpay.slack.com/archives/C05RBSKC6KD/p1706089761213119?thread_ts=1706083683.738819&cid=C05RBSKC6KD

I am considering this comment as addressed